test/integration/sagemaker/test_distributed_operations.py

```diff
@@ -44,11 +44,8 @@ def test_dist_operations_multi_gpu(sagemaker_session, ecr_image, dist_gpu_backen
 
 
 @pytest.mark.skip_cpu
-def test_dist_operations_fastai_gpu(sagemaker_session, ecr_image, py_version):
-    if py_version != PYTHON3:
-        print('Skipping the test because fastai supports >= Python 3.6.')
-        return
-
+@pytest.mark.skip_py2
```

Comment:
You should skip both CPU and PY2 here.

Reply:
This test already skips cpu on the line above